shell/platform/darwin/ios/platform_message_handler_ios_test.mm

```diff
@@ -12,9 +12,6 @@
 #import "flutter/lib/ui/window/platform_message.h"
 #import "flutter/lib/ui/window/platform_message_response.h"
 #import "flutter/shell/common/thread_host.h"
-#import "flutter/shell/platform/darwin/common/framework/Headers/FlutterMacros.h"
-
-FLUTTER_ASSERT_NOT_ARC
 
```

Comment:
FLUTTER_ASSERT_ARC? Even though nothing had to change, it could get code added in the future that assumes ARC.